src/pages/Contato/Contato.page.jsx

```diff
@@ -1,15 +1,178 @@
 import React from "react";
-import "./contatoPage.scss"
-const ContatoPage = () => {
-    return (
-        <div className="contacts-container">
-            <div className="contacts-header">
-                <div className="contacts-header-title-small">RESPOSTAS EM ATÉ 24H</div>
-                <div className="contacts-header-title">nosso contato</div>
+import { useState } from "react";
+import "./contatoPage.scss";
+import { ContactCard } from "../../components/ContactCard/ContactCard.jsx";
+
+import facebook from "../../components/img/Icons/facebook.png";
+import instagram from "../../components/img/Icons/instagram.png";
+import map from "../../components/img/Icons/map.png";
+import messageImg from "../../components/img/Icons/message.png";
+import phoneImg from "../../components/img/Icons/phone.png";
+import youtube from "../../components/img/Icons/youtube.png";
+
+import dec from "../../components/img/dec.svg";
+import rio from "../../components/img/rio.png";
+import sp from "../../components/img/sp.png";
+import Footer from "../../components/footer/Footer";
+
+export const ContatoPage = () => {
+  const [name, setName] = useState("");
+  const [phone, setPhone] = useState("");
+  const [email, setEmail] = useState("");
+  const [message, setMessage] = useState("");
+  const [isFormValid, setIsFormValid] = useState(false);
+
+  function handleNameChange(event) {
+    console.log(event.target.value);
+    setName(event.target.value);
+  }
+
+  function handlePhoneChange(event) {
+    console.log(event.target.value);
+    setPhone(event.target.value);
+  }
+
+  function handleEmailChange(event) {
+    console.log(event.target.value);
+    setEmail(event.target.value);
+  }
+
+  function handleMessageChange(event) {
+    console.log(event.target.value);
+    setMessage(event.target.value);
+  }
```

Comment:
can u create one handler change for all inputs?

Reply:
ok